tests/unittest/_torch/visual_gen/test_attention_perf.py

```
@@ -834,6 +853,191 @@ def test_fa4_cross_attn_quick(
+    All tests use WAN model dimensions (head_dim=128, num_heads=12) because the
+    SM100 SageAttention cubins are generated for head_dim=128 only.
+    """
+
+    # WAN-realistic seq_lens (VAE 8x spatial, 4x temporal, patch [1,2,2])
+    WAN_SEQ_LENS = [
+        (1, 12, 14040, 128, "wan_1.3b_480p_33f"),
+        (1, 12, 32760, 128, "wan_1.3b_480p_81f"),
+        (1, 40, 14040, 128, "wan_14b_480p_33f"),
+        (1, 40, 32760, 128, "wan_14b_480p_81f"),
+        (1, 40, 75600, 128, "wan_14b_720p_81f"),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)